app/assets/stylesheets/forever_style_guide/modules/_nav-dropdowns.scss

```diff
@@ -68,6 +77,10 @@ $nav-icon-width: 60px;
     @include active-style($color-accent);
   }
 }
+//ugly cousin active state
```

Comment:
i still think we should namespace this as navbar-emphasized or something like that rather than explicitly calling it deals since we could conceivably apply that style to other items. ANd then it would be best to include that in a standalone style module.